### ISSUE TITLE
Photos uploaded but not found - Neutron 4.4.0.

### ISSUE THREAD

**Issue by akvaskov** [moxi9/phpfox#1591]
Hi,

I uploaded several photos. They appear in the "Photos" block on my page but when I try to "View more photos", nothing found on a redirected page. Where are my photos? :)
![photos uploaded but not found](https://cloud.githubusercontent.com/assets/18528411/17204542/c8234e4e-54ae-11e6-87c2-3407bd5449f9.png)

**Comment by Data66PF** [comment]
https://v4.phpfox.com/Data66/ cannot reproduce. Perhaps you have third party products interfering.

**Comment by SalmanKhan86** [comment]
Same Problem here. i have Uploaded Many photos but when i go to view more photos Nothing found.where are my photos? i am using latest stable 4.4.0
![photo problem](https://cloud.githubusercontent.com/assets/13433124/17248737/01ef2ef2-55ba-11e6-80a5-b596f70c08a3.png)
![photo not found](https://cloud.githubusercontent.com/assets/13433124/17248739/0a712b98-55ba-11e6-88c5-735a0c83b494.png)

**Comment by akvaskov** [comment]
As I understood, it depends on a way how you upload photos. If they are added to the feed, you will never see them on the photos page. But in the photos block they will appear. From my point of view, that's really strange. But from point of view of the developers, it works as designed.

**Comment by akvaskov** [comment]
Amen! :)

**Comment by TrueAmericans** [comment]
Yes, I'm also having this problem... 2 of the same photos from the old version show, but none of the other dozens of old ones or any added since upgrading to 4.4.0 Stable...

**Comment by Data66PF** [comment]
Photos added to the feed are comments. That part of this is working as it has always worked. However, the view more might be an issue.

**Comment by akvaskov** [comment]
Yeah, let's try to solve this issue. But furthermore, it was proposed several times to redesign the engine and to make it possible to upload photos from the feed to a separate album.

**Comment by Data66PF** [comment]
We will not address feature requests here as we said.

**Comment by Data66PF** [comment]
Tested the "view more" issue and cannot reproduce any issue at the demo. I loaded 11 photos to the feed. They show in my block on my profile. If I click view more, it goes to my photo section of my profile and does not have any error. It does not show the photos there as it is not designed to do that for comments which is what these are. We do have a feature request https://github.com/moxi9/phpfox-feature-requests/issues/110 which we've marked as Interested for looking at when we are looking at features for versions above 4.5.
